gapic-generator-java/src/test/proto/explicit_dynamic_routing_header_testing.proto

```diff
@@ -265,9 +266,58 @@ service ExplicitDynamicRoutingHeaderTesting {
       }
     };
   }
+
+  // This method echoes the ComplianceData request. This method exercises
+  // including a non-proto3-optional enum field in the URL path while sending the
+  // entire request object in the REST body.
+  rpc RepeatDataPathEnum(RepeatRequest) returns (RepeatResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/repeat/kingdom/{info.f_kingdom}:pathenum"
```

Comment:
This is for implicit routing header, can you please add another test case for explicit routing header?